implementations/rust/ockam/ockam_abac/src/mem.rs

```diff
@@ -73,6 +84,10 @@ impl PolicyStorage for Memory {
         self.inner.write().unwrap().set_policy(r, a, p);
         Ok(())
     }
+
+    async fn policies(&self, r: &Resource) -> Result<Vec<(Action, Expr)>> {
+        Ok(self.inner.write().unwrap().policies(r))
```

Comment:
Given that these methods return a `Result`, why don't handle the errors with `?`? AFAIK, we do these unwraps because we are using the std locks and they are usually used like this, but is there any specific reason for not bubbling up the errors in this case?